src/webhooks/IWebhookProvider.ts

```diff
@@ -7,9 +7,8 @@ export interface IWebhookProvider {
      * Called whenever the publically accessible url for this Rocketlet is called,
      * if you handle the methods differently then split it out so your code doesn't get too big.
      *
-     * @param method the method this was called with (GET, POST, etc)
      * @param request the actual request made
      * @return the response to send to the client
      */
-    webhook_event(method: RequestMethod, request: IRequest): IResponse;
+    webhook_event(request: IRequest): IResponse;
```

Comment:
Can we maybe rename this method to be `executeWebhook`? Since `webhook_event` is now extremely different from the other methods.